datadog/resource_datadog_monitor_test.go

```diff
@@ -49,11 +49,13 @@ func TestAccDatadogMonitor_Basic(t *testing.T) {
 						"datadog_monitor.foo", "require_full_window", "true"),
 					resource.TestCheckResourceAttr(
 						"datadog_monitor.foo", "locked", "false"),
-					// Tags are sorted
+					// Tags are a TypeSet => use a weird way to access members by their hash
 					resource.TestCheckResourceAttr(
-						"datadog_monitor.foo", "tags.0", "baz"),
+						"datadog_monitor.foo", "tags.#", "2"),
```

Comment:
This just means there are two tags, right?

Reply:
Correct, this checks that the number of tags is what we expect it to be, so two.